orchagent/aclorch.cpp

```diff
@@ -288,6 +292,17 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
             value.aclfield.data.u16 = to_uint<uint16_t>(attr_value);
             value.aclfield.mask.u16 = 0xFFFF;
         }
+        else if (attr_name == MATCH_VLAN_ID)
+        {
+            value.aclfield.data.u16 = to_uint<uint16_t>(attr_value);
+            value.aclfield.mask.u16 = 0xFFF;
```

Comment:
Should this be 0xFFFF?

Reply:
4095 == 0xFFF

Reply:
Exactly what @qiluo-msft said, VLAN ID is a 12 bit field so we need a 12 bit mask for it.